DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -4,55 +4,126 @@
  * Copyright 2019-Present Datadog, Inc.
  */
 
+import Foundation
 import DatadogInternal
 
-/// Defines methods to send WebView related information
-internal protocol WebViewTracking {
-    /// Sends a message
-    /// - Parameter message: The message to send
-    func send(message: WebViewTrackingMessage) throws
-}
+#if !os(tvOS)
```

Comment:
question

seems like `WebViewTracking` should not have tvOS as supported platform. If true, please make sure we are not enabling it accidentally.

Reply:
Yes, I didn't want to change current binary support but I think the entire `DatadogWebViewTracking` shouldnt be built for tvOS.